packages/xod-project/src/constants.js

```diff
@@ -6,6 +6,8 @@
  * @enum {string}
  */
 export const ERROR = {
+  // dead references
+  DEAD_REFERENCES_FOUND: 'Patch "{patchPath}" contains dead references:\n{message}\nFix or delete them to continue.',
```

Comment:
That’s what I talked about. Not i18n friendly. Should be:

```javascript
export const formatDeadReferencesFound = (patchPath, submessage) => [
  `Patch ${patchPath} contains dead references:`
  submessage,
  'Fix or delete them to continue.',
].join('\n');
```

It’s not a constant. So it should live in `messages.js`. I suggest moving it there and leave a `// TODO:` about legacy for existing “constants”.